src/main/java/uk/gov/justice/digital/hmpps/csr/api/controllers/DetailController.kt

```diff
@@ -29,15 +37,51 @@ class DetailController(private val detailService: DetailService) {
     return ResponseEntity.ok(details)
   }
 
-  @Operation(summary = "Retrieve all modified shifts for a plan unit")
-  @GetMapping("planUnit/{planUnit}/shifts/updated")
+  @Operation(summary = "Retrieve all modified shifts for a plan unit", deprecated = true)
```

Comment:
would expect this to also be deprecated in kotlin with the `@Deprecated`  - might automatically add `deprecated = true` to the endpoint...